README.md

```diff
@@ -60,5 +60,70 @@ To register a device, run as root from LmP device:
 (i.e. `sudo DEVICE_API=http://10.0.2.2:80/sign lmp-device-register [-T <TOKEN>] [-f <factory>]`)
 
 # Testing/Troubleshooting
-This project includes a simple `fake-lmp-device-register` that can be used
-to quickly test this project's API.
+
+In scenarios where you'd like to test the API call without involving a physical device or emulation:
+
+This project includes a simple [fake-lmp-device-register](./fake-lmp-device-register) tool. 
+This tool mimics the behavior of `lmp-device-register`, enabling you to quickly test this project's API 
+without a real device.
+
+## Running the fake-lmp-device-register
+
+### Dependencies:
+- Python 3
+- `requests` library
+- `openssl` command-line tool
+- `curl`
+
+### Usage:
+
+1. Install the required Python library using pip:
+
+```shell
+pip3 install requests
+```
+
+2. Execute the fake-lmp-device-register tool:
+```shell
+python3 fake-lmp-device-register --registration-url "http://<IP of docker-compose host>:80/sign" --factory <factory>
+```
+
+3. Verify Connectivity with the Server:
+
+After successfully executing the fake-lmp-device-register, a new directory, `var/sota`, will be created in the current 
+directory. This contains a curl script and the necessary certificates.
```

Comment:
We don't create this directory; and the user may opt to have a different directory.

I'd rather have the command above changed to use the `--sota-dir ${SOTA_DIR}`. 
And tell in paragraph 2 that this command creates fake device config and certificates in that directory.